bytecomp/bytelink.ml

```diff
@@ -19,6 +19,8 @@ open Misc
 open Config
 open Cmo_format
 
+module Idents = Symtable.GlobalIds
```

Comment:
I don't understand the choice of using `Idents` here. If I had to guess what this module is about, it would be "lists/bags of Ident.t". 

There are other related naming choices in this patch that I don't understand:

- you use the type `global` but a module named `GlobalIds`; either we think of the values as "a global", or "the identifier for a global", both are valid naming choices, but please use a consistent choice for the type and the module name (so `global_id` and `Global_id`, or `global` and `Global`)
- as suggested above, I don't understand the plural in `GlobalIds`; the usual convention is to use a singular for the module name (`Array`, `Option`, not `Arrays`, `Options`). (I would also expect the type `Global` or `Global_id` or `GlobalId` to export a type component `type t = global_id`, but not having this is less confusing than the unusual naming convention).

To summarize, I would expect `module Global_id = Symtable.Global_id` or `module Global = Symtable.Global` or `module GlobalId = Symtable.GlobalId` here, with a consistent naming choice for the type itself. (Or instead of having a separate type, the definition could be the definition of `Global_id.t` directly.)